src/page/Home.tsx

```diff
@@ -265,7 +202,25 @@ class Home extends React.Component<{ history: any }> {
                     </Row>
                     <Row className="mb-5">
                         <Col md={10} className="mx-auto text-center">
-                            {searchExample}
+                            <SearchExample genomeBuild={this.genomeBuild} />
+                            <div style={{ color: 'gray', fontSize: '14px' }}>
+                                {`Genome build: ${this.genomeBuild}`}
+                                <a
+                                    href={
+                                        this.genomeBuild === GENOME_BUILD.GRCh37
+                                            ? 'https://grch38.genomenexus.org'
+                                            : 'https://www.genomenexus.org'
```

Comment:
we should think a bit about how to manage these URLs. If someone tries to deploy this locally they prolly don't want to point to our production instances. Maybe we should only enable these links based on some configuration or something

Reply:
not a priority tho but maybe good to file a ticket indicating some places where we are directly pointing to our production instances rather than the root domain